api/runnerpool/placer_tracker.go

```diff
@@ -113,7 +113,24 @@ func (tr *placerTracker) HandleDone() {
 
 // RetryAllBackoff blocks until it is time to try the runner list again. Returns
 // false if the placer should stop trying.
-func (tr *placerTracker) RetryAllBackoff(numOfRunners int) bool {
+func (tr *placerTracker) RetryAllBackoff(numOfRunners int, err error) bool {
+
+	// If the last call to obtain runners failed due to a user error or
+	// misconfiguration then do not re-attempt to obtain a runner.
+	if err != nil {
+		tr.HandleFindRunnersFailure(err)
```

Comment:
This'll be called if and when we exit the Placer loop - I don't think you want to do it here as well.

Reply:
Thanks. You are right, I think was seeing double logs in Kibana. Removed it.

Reply:
I still see this in the code, did you push the latest version of your change?